src/Impostor.Server/Recorder/PacketRecorder.cs

```diff
@@ -35,17 +39,32 @@ public PacketRecorder(ILogger<PacketRecorder> logger, IOptions<DebugConfig> opti
 
             _channel = Channel.CreateUnbounded<byte[]>(new UnboundedChannelOptions
             {
-                SingleReader = true,
-                SingleWriter = false,
+                SingleReader = true, SingleWriter = false,
```

Comment:
Keep on separate lines.

Reply:
Blame your .editorconfig, will fix it

Reply:
Can you check if code format does the same thing on your rider? I can't force it to not do that, pretty annoying.

Reply:
Does the same for me